### PR TITLE
Fixed case for persons names, events and groups titles

### PR DESCRIPTION
**Pull request by HeahDude**

### CODE REVIEW

File: src/Form/TypeExtension/TextTypeExtension.php [hunk 1]
```diff
@@ -53,17 +68,56 @@ public function getExtendedType()
...
+    public static function formatDataAsIdentity(FormEvent $event): void
+    {
+        $data = $event->getData();
+
+        if (\is_string($data) && '' !== $data) {
+            $event->setData(self::formatIdentityCase($data));
+        }
+    }
+
+    public static function formatTitleCase(string $string)
```
Comment (Remg) [inline src/Form/TypeExtension/TextTypeExtension.php]: missing return type

File: src/Form/TypeExtension/TextTypeExtension.php [hunk 2]
```diff
@@ -53,17 +68,56 @@ public function getExtendedType()
...
+    {
+        return \mb_convert_case($string, MB_CASE_TITLE, 'UTF-8');
+    }
+
+    public static function formatIdentityCase(string $string)
```
Comment (Remg) [inline src/Form/TypeExtension/TextTypeExtension.php]: missing return type

File: src/Form/TypeExtension/TextTypeExtension.php (hunk 1 above)
Comment (HeahDude) [reply src/Form/TypeExtension/TextTypeExtension.php]: Fixed, thx!

File: src/Form/TypeExtension/TextTypeExtension.php (hunk 2 above)
Comment (HeahDude) [reply src/Form/TypeExtension/TextTypeExtension.php]: Fixed

**Pull request merged**